package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "deployment-tracker",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
I'm not sure if it's necessary to bump the version number. We should probably just get rid of `version` here as we're doing continuous deployment.